hw1/v1tal4ik.js

```diff
@@ -0,0 +1,173 @@
+// 1.Write a Javascript program to create a new string which is n copies of a given string where n is a non-negative integer.
+
+function repeatString(str = '', value = 0) {
+  if ((typeof (str)) !== 'string' || (typeof (value) !== 'number') || value < 0) {
+    const error = 'Error: type of arguments are invalid';
+    console.error(error);
+    return error;
+  }
+
+  let basicStr = str;
+  for (let i = 1; i < value; i++) {
+    basicStr += str;
+  }
+  return basicStr;
+}
+
+const repeatStr = repeatString('Hi Tomas!', 3);
+console.log({ repeatStr });
+
+
+// 2.Write a Javascript program to check if a string starts with "if".
+
+function checkIfStartsWithIf(str = '') {
```

Comment:
there is a simpler solution but this one demostrates that you know what regExp is 😉